Dockerfile

```diff
@@ -304,25 +304,23 @@ RUN pip install bleach && \
     pip install decorator && \
     pip install entrypoints && \
     pip install html5lib && \
-    # Latest version breaks nbconvert: https://github.com/ipython/ipykernel/issues/422
-    pip install ipykernel==5.1.1 && \
+    pip install ipykernel && \
     pip install ipython && \
     pip install ipython-genutils && \
     pip install ipywidgets && \
     pip install isoweek && \
-    # Lastest version breaks ipython. https://github.com/ipython/ipython/issues/12740
-    pip install jedi==0.17.2 && \
```

Comment:
JupyterLab does call out to jedi, do we know this is resolved too?

Reply:
Yup it works: https://screenshot.googleplex.com/95HSLyJ4RMv7fos